src/mscorlib/shared/System/Text/ValueStringBuilder.cs

```diff
@@ -58,7 +70,20 @@ public override string ToString()
             return s;
         }
 
-        public ReadOnlySpan<char> AsSpan() => _chars.Slice(0, _pos);
+        /// <summary>
+        /// Returns a span around the contents of the builder.
+        /// </summary>
+        /// <param name="terminate">Ensures that the builder has a null char after <see cref="Length"/></param>
+        public ReadOnlySpan<char> AsSpan(bool terminate = false)
```

Comment:
Is this going to make the method go over the inlining threshold?

Reply:
I don't know the full rules. I can split/attribute?

Reply:
Perfview can show you whether something was innings and if not why not.